src/backend/distributed/executor/multi_utility.c

```diff
@@ -384,6 +375,18 @@ multi_ProcessUtility(Node *parsetree,
 		SetUserIdAndSecContext(savedUserId, savedSecurityContext);
 	}
 
+	if (ddlJobs != NIL)
```

Comment:
Maybe a comment which notes we run the ddl on the coordinator above with standard process utility and now we will run on worker nodes.